docs/integrations/send-logs-to-elasticsearch.md

```diff
@@ -20,43 +21,41 @@ Collect these values for the connection:
 Start by setting up an external service integration for Elasticsearch.
 
 1.  Log in to the [Aiven Console](https://console.aiven.io/).
-2.  Navigate to **Service Integration** from the menu on the left.
-3.  You'll see a list of external services you can integrate with
-    Aiven.
-4.  Select **External Elasticsearch** from the list.
-5.  Select **Add new endpoint**.
-6.  Set a preferred *endpoint name*, we'll call it `CONNECTION_NAME`
+1.  Go to **Service Integration** from the menu on the left.
+1.  Select **External Elasticsearch** from the list.
+1.  Select **Add new endpoint**.
+1.  Set a preferred *endpoint name*, we'll call it `CONNECTION_NAME`
     later.
-7.  In the connection URL field set the connection string in a format
+1.  In the connection URL field set the connection string in a format
     `https://ELASTICSEARCH_USER:ELASTICSEARCH_PASSWORD@ELASTICSEARCH_HOST:ELASTICSEARCH_PORT`,
     using your own values for those parameters.
-8.  Set desired index prefix, that doesn't overlap with any of already
+1.  Set desired index prefix, that doesn't overlap with any of already
     existing indexes in your Elasticsearch service.
-9.  If you need a certificate to access the endpoint, add the body of
+1.  If you need a certificate to access the endpoint, add the body of
     your CA certificate in PEM format. This field is optional.
-10. Set other fields based on your requirements, or leave the default
+1.  Set other fields based on your requirements, or leave the default
     values there.
-11. Select **Create**.
+1.  Select **Create**.
 
 A new service integration will be added. You can now reference your
 service by the `CONNECTION_NAME` you chose.
 
 ## Send logs to an external service
```

Comment:
Not sure, but perhaps "Send service logs to Elasticsearch" would be better when someone's searching the docs? The reason being that there are different types of logs in Aiven, and there's no longer an "internal" Elasticsearch service in Aiven.